test/acceptance/test/add_tests.go

```diff
@@ -260,13 +210,14 @@ var _ = Describe("Weave GitOps Add App Tests", func() {
 		})
 
 		By("And repos created have private visibility", func() {
-			if os.Getenv("GIT_PROVIDER") != "gitlab" {
-				Expect(getGitRepoVisibility(gitOrg, tip.appRepoName, gitProvider)).Should(ContainSubstring("private"))
-			}
+			Expect(getGitRepoVisibility(gitOrg, tip.appRepoName, gitProvider)).Should(ContainSubstring("private"))
 		})
 
-		By("When I remove an app", func() {
+		By("When I remove the app", func() {
```

Comment:
did you want this to change _everywhere_?

Reply:
> did you want this to change everywhere?

Nah, definitely too much work for the scope of this PR! I figured I'd just raise it since you were refactoring anyway. Feel free to decide how many you want to change :smile: